Core/Python/add_device_to_static_group.py

```diff
@@ -20,33 +20,158 @@
 #
 """
 SYNOPSIS:
-   Add a host to a static group
+   Add one or more hosts to an existing static group.
 
 DESCRIPTION:
-   This script exercises the OME REST API to create a new static
-   group. The user is responsible for adding devices to the
-   group once the group has been successfully created.
-   For authentication X-Auth is used over Basic Authentication
-   Note that the credentials entered are not stored to disk.
+   This script exercises the OME REST API to add one or more
+   hosts to an existing static group. For authentication X-Auth
+   is used over Basic Authentication. Note: The credentials entered
+   are not stored to disk.
 
 EXAMPLE:
-   python create_static_group.py --ip <xx> --user <username>
-        --password <pwd> --groupname "Random Test Group" --devicename "cmc1"
+   python add_device_to_static_group.py --ip <xx> --user <username>
+        --password <pwd> --groupname "Random Test Group" --devicenames "cmc1,host3,192.168.1.5"
 """
 
 import json
-import sys
 import argparse
 from argparse import RawTextHelpFormatter
 import urllib3
 import requests
 
 
-def add_device_to_static_group(ome_ip_address, ome_username, ome_password, group_name):
-    """ Authenticate with OME and enumerate groups """
+def get_device_list(ome_ip_address: str, headers: dict) -> list:
```

Comment:
This method is referenced only once - and the calling method then has to assemble the dictionary of service tags to id - you could just return that dictionary from this method

Reply:
Updated